ansible/upgrade_sonic.yml

```diff
@@ -49,6 +49,16 @@
         - fail: msg="image_url is not defined"
           when: image_url is not defined
 
+        - name: define retain_copies if not defined
+          set_fact:
+            retain_copies: 2
+          when: retain_copies is not defined
```

Comment:
I feel the word "copies" is a bit confusing. Something along the lines of `max_images_to_retain` (but less verbose) is a more accurate name.